### ISSUE TITLE
CGPR1 motion sensor is broken in 5.3.0

### ISSUE THREAD

**Issue by andrey-yantsen** [custom-components/ble_monitor#511]
I'm seeing the following error in the logs after upgrading:

```
2021-09-14 15:00:52 ERROR (Thread-3) [homeassistant] Error doing job: Fatal error: protocol.data_received() call failed.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/asyncio/selector_events.py", line 870, in _read_ready__data_received
    self._protocol.data_received(data)
  File "/usr/local/lib/python3.9/site-packages/aioblescan/aioblescan.py", line 1852, in data_received
    self.process(packet)
  File "/config/custom_components/ble_monitor/__init__.py", line 567, in process_hci_events
    sensor_msg, tracker_msg = self.ble_parser.parse_data(data)
  File "/config/custom_components/ble_monitor/ble_parser/__init__.py", line 88, in parse_data
    sensor_data = parse_xiaomi(self, adstruct, mac, rssi)
  File "/config/custom_components/ble_monitor/ble_parser/xiaomi.py", line 763, in parse_xiaomi
    result.update(resfunc(object))
TypeError: obj000f() missing 1 required positional argument: 'device_type'
```

**Comment by Ernst79** [comment]
Could you try changing the following line in `custom_components/ble_monitor/bleparser/xiaomi.py`, line 760

`                    if hex(obj_typecode) in ["0x1001", "0x000F"]:`

to 

`                    if hex(obj_typecode) in ["0x1001", "0x000F", "0x000f"]:`

**Comment by andrey-yantsen** [comment]
@Ernst79 yup, that worked. It seems like `hex()` is always returning a lowercase value.

**Comment by andrey-yantsen** [comment]
Noooope, it didn't worked, sorry :\ Probably I was just somehow lucky and got another message.

Anyway, changing `0x000F` to `0xf` has helped.

**Comment by Ernst79** [comment]
Thanks, Fixed in 5.3.1.
